### PR TITLE
rip out configure-aspen.py

### LINKED ISSUE #373: do away with configure-aspen.py
Now that we're a framework and not an app server, we don't need a special way to exec python code to configure your site. Just use Python! This means that if want to use Python to configure your site you'll need to unroll one of the following, depending on your deployment scenario:

 - `aspen/wsgi.py`
 - `aspen/__main__.py`
 - `fcgi_aspen.py`

[call](https://www.youtube.com/watch?v=1kKh2URbJvs)

### PR DESCRIPTION
**Pull request by whit537**
Closes #373.

### CODE REVIEW

**Comment by whit537** [comment]
Rebased on master.

**Comment by whit537** [comment]
Next step on this is to fix up the test suite.

`============== 13 failed, 958 passed, 1 skipped in 24.56 seconds ===============`

**Comment by whit537** [comment]
Rebased on master.

**Comment by techtonik** [comment]
Is it already gone? https://github.com/gratipay/inside.gratipay.com/pull/263#issuecomment-113510235

What is the replacement?

**Comment by whit537** [comment]
I don't believe it's already gone, but the replacement is to make a plain ol' Python module.

**Comment by techtonik** [comment]
@whit537 ok, I've checked that it is imported, but I still don't why the stacktrace? https://github.com/gratipay/inside.gratipay.com/pull/263#issuecomment-113509846

**Comment by whit537** [comment]
Ready for review, @techtonik @pjz et al.

**Comment by whit537** [comment]
(Also, re-re-rebased on master.)

File: doc/project_root/index.html.spt [hunk 1]
```diff
@@ -22,12 +22,8 @@ flat.</p>
 
 <h4>configuration files</h4>
 
-<p>Aspen will read the following files from <code>project_root</code>.</p>
-
-<ul>
-    <li><code><a href="/configure-aspen.py/">configure-aspen.py</a></code></li>
-    <li><code><a href="/mime.types/">mime.types</a></code></li>
-</ul>
+<p>Aspen will read the <code><a href="/mime.types/">mime.types</a></code> file
```
Comment (pjz) [inline doc/project_root/index.html.spt]: (if any)

**Comment by techtonik** [comment]
Need a note in Changelog.md about the removal and intended replacement.

**Comment by whit537** [comment]
Good catch. Done in https://github.com/gratipay/aspen-python/commit/579261b8dabdbbae59d8c63f6b077acc45ae5c54.

**Comment by techtonik** [comment]
!m @whit537

**Pull request merged**